tasks/test-metadata/0.2/test-metadata.yaml

```diff
@@ -106,3 +114,28 @@ spec:
 
         # Write job-spec to result file
         echo -n "$JOB_SPEC" > $(results.job-spec.path)
+
+        if [ "$ENABLE_PREFETCH" == "true" ]; then
+
+          cosign download attestation "$COMPONENT_CONTAINER_IMAGE" > cosign_metadata.json
+
```

Comment:
COMPONENT_SOURCE_ARTIFACT="$(jq -r \
            '.payload | @base64d | fromjson | .predicate.buildConfig.tasks[] |
            select(.name == "prefetch-dependencies") | .results[] | select(.name == "CACHI2_ARTIFACT") | .value' \
            cosign_metadata.json)"
            
            I think we can directly get the result of the build task directly and not use oras here to get the sha etc. Will be more accurate in this case using the results

Reply:
Ah, using the build task results is way better. Thanks for the suggestion, already fixed it!